integrationtest/wildfly/src/wildflyConfig/standalone/configuration/standalone-full-testqueues.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" ?>
```

Comment:
That's right.
The changes are also quite smaller than what they used to be. Essentially we have:
 - 2 seconds TX default timeout
 - debug logging enabled for `org.hibernate.search`